### PR TITLE
Update Google Cloud project-setup.md

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @joeliedtke @jlewi @8bitmp3

### CODE REVIEW

File: content/en/docs/gke/deploy/project-setup.md [hunk 1]
```diff
@@ -54,23 +54,23 @@ Follow these steps to set up your GCP project:
...
+
+    Refer to [Anthos Service Mesh documentation](https://cloud.google.com/service-mesh/docs/archive/1.4/docs/gke-install-new-cluster#setting_credentials_and_permissions) for details.
 
 You do not need a running GKE cluster. The deployment process creates a
 cluster for you.
 
 ## Next steps
 
-* [Set up an OAuth credential](/docs/gke/deploy/oauth-setup) if you want to use 
+* [Set up an OAuth credential](/docs/gke/deploy/oauth-setup) to use
```
Comment (Bobgy) [inline content/en/docs/gke/deploy/project-setup.md:72]: this is now a required step

File: content/en/docs/gke/deploy/project-setup.md [hunk 2]
```diff
@@ -28,7 +28,6 @@ Follow these steps to set up your GCP project:
     * [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
     * [Kubernetes Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com)
     * [Identity and Access Management (IAM) API](https://console.cloud.google.com/apis/library/iam.googleapis.com)
-    * [Deployment Manager API](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
     * [Service Management API](https://console.cloud.google.com/apis/api/servicemanagement.googleapis.com)
     * [Cloud Resource Manager API](https://console.developers.google.com/apis/library/cloudresourcemanager.googleapis.com)
     * [Cloud Filestore API](https://console.developers.google.com/apis/library/file.googleapis.com)
```
Comment (jlewi) [inline content/en/docs/gke/deploy/project-setup.md:33]: We don't really use Cloud FileStore so you could probably get rid of this.

File: content/en/docs/gke/deploy/project-setup.md [hunk 3]
```diff
@@ -54,23 +54,23 @@ Follow these steps to set up your GCP project:
   to learn how to check your quota and how to request an increase in quota.
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply content/en/docs/gke/deploy/project-setup.md:53]: so far the title has been consistently using `set up ...`, `delete ...`, `deploy ...`, so I changed to `Set up a Google Cloud Project` at least for now.

[the remark this answers is not in the document]
Comment (Bobgy) [reply content/en/docs/gke/deploy/project-setup.md:53]: Yes, I think the CLI way helps too, let me add it there.

File: content/en/docs/gke/deploy/project-setup.md [hunk 4]
```diff
@@ -54,23 +54,23 @@ Follow these steps to set up your GCP project:
...
   
 1. Initialize your project to ready it for Anthos Service Mesh installation.
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply content/en/docs/gke/deploy/project-setup.md:55]: It's enabled programmatically in later stages from resource definition in https://github.com/kubeflow/manifests/blob/v1.1-branch/gcp/v2/cnrm/gcp-services.yaml

**Comment by Bobgy** [comment]
Thank you for the suggestions @8bitmp3 @jlewi !

I'll get this in first, if there are further comments, feel free to raise them. I'll follow up in later PRs when necessary.

**Pull request merged**